### ISSUE TITLE
Extremely slow build (iOS) after adding Firebase

### ISSUE THREAD

**Issue by vpiskunov** [EddyVerbruggen/nativescript-plugin-firebase#1401]
### After adding the plugin, the build becomes painfully slow (between 15 to 25 minutes)

This happens on every `tns run ios` and it does `xcode build` during that. 

There is a lot of **warnings about Protobuf**, amongst others.

Before adding this plugin, everything is fine - build happens in **1-2 minutes**.

I have experimented with reinstalling the plugin, reinstalling platforms, cleaning platforms, as well as tested this on multiple projects which use NativeScript 5.4, 6.1.0, 6.1.1 etc. - but nothing helps.

Also tried using different versions of Node & npm - same results.

Really struggling to develop with this...

Logs are here:
https://gist.github.com/vpiskunov/69cc91d1352f3f02486236997c44b4f8

Appreciate any help @EddyVerbruggen!

**Comment by zghotlawala** [comment]
Yeah why they don't resolve this major issue!!!! While building my application almost hangs the whole system!!!

**Comment by vpiskunov** [comment]
@zghotlawala can you please try cloning & installing & running this project https://github.com/EddyVerbruggen/footplr

And try running your own project’s build after you do that - let us know if that helps resolve it? It’s a weird but potential workaround that may have an impact.

**Comment by kdiehl** [comment]
This seems to relate to:
https://github.com/firebase/firebase-ios-sdk/issues/2665

**Comment by EddyVerbruggen** [comment]
I'd love to do something about it, but how the _beep_ should I be able to change this behavior?

**Comment by foxted** [comment]
@EddyVerbruggen Maybe updating the dependencies can solve this issue? https://github.com/firebase/firebase-ios-sdk/issues/2665#issuecomment-537983557

**Comment by foxted** [comment]
Seems to be coming for these processes created while building the application, I believe they are related to Crashalytics

![Screen Shot 2019-10-16 at 7 42 56 AM](https://user-images.githubusercontent.com/1385160/66929910-adde7800-efe8-11e9-99c8-ed5ad405403d.png)

**Comment by guigingras** [comment]
Same issue here.

**Comment by alqabali** [comment]
same problem here! and I use also crashlytics as @foxted  mentioned

**Comment by dtomaszewski** [comment]
same here unfortunately :(
